starter_code/src/movies.js

```diff
@@ -116,5 +114,19 @@ dramaMoviesRate(movies);
 
 // Iteration 6: Time Format - Turn duration of the movies from hours to minutes
 
+function turnHoursToMinutes(arr) {
```

Comment:
The tests ask you to return a new array, so here you could use the map() method to loop over the original array, since this method creates a new array.